### PR TITLE
chore: add enable Cloud Init steps

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by apeabody** [comment]
@Shabirmean

File: extras/anthos-vmruntime/README.md [hunk 1]
```diff
@@ -54,11 +54,13 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
     ```
 
 - Make the bucket **publicly readable**
+    > NOTE: For typical use the bucket should be **private** and [credentials](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/vm-runtime/create-storage-credentials) used to import the image into VM Runtime.
```
Comment (Shabirmean) [inline extras/anthos-vmruntime/README.md:57]: I am not sure I understand this sentence 🤔 

image

Is it trying to say:

`.... the bucket and the credentials should be provate?`

Comment (Shabirmean) [reply extras/anthos-vmruntime/README.md:57]: Can we format the `NOTE` as follows:

`> **Note**:`

> **Note:**

File: extras/anthos-vmruntime/README.md [hunk 2]
```diff
@@ -67,6 +69,26 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
...
+
+- Enable Cloud Init using virt-sysprep
+    ```sh
+    virt-sysprep -a pos-vm.qcow2 \
+        --uninstall google-compute-engine,google-compute-engine-oslogin,google-guest-agent,google-osconfig-agent \
+        --delete '/etc/cloud/cloud.cfg.d/*.cfg' \
+        --mkdir /etc/cloud/cloud.cfg.d \
+        --write /etc/cloud/cloud.cfg.d/10_anthos.cfg:'datasource_list: [ NoCloud, ConfigDrive, None ]\n'
+    ```
+    > NOTE: In many recent Linux distributions `virt-sysprep` is part of the `libguestfs-tools` package.
```
Comment (Shabirmean) [inline extras/anthos-vmruntime/README.md:85]: Can we format the `NOTE` as mentioned above?

**Review by Shabirmean — COMMENTED** [review]
Some minor comments.

Thank you @apeabody

File: extras/anthos-vmruntime/README.md (hunk 2 above)
Comment (apeabody) [reply extras/anthos-vmruntime/README.md]: SG

File: extras/anthos-vmruntime/README.md (hunk 1 above)
Comment (apeabody) [reply extras/anthos-vmruntime/README.md]: I rewrote to hopefully be clearer.  The GCS bucket should be private.  Then credentials can be configured in VM Runtime to access the image from the private bucket.

File: extras/anthos-vmruntime/README.md [hunk 3]
```diff
@@ -54,11 +54,13 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
     ```
 
 - Make the bucket **publicly readable**
+    > **Note**: For typical use the bucket should be **private**, and [credentials](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/vm-runtime/create-storage-credentials) added in VM Runtime to securely access the private image.
```
Comment (Shabirmean) [inline extras/anthos-vmruntime/README.md:57]: **(Nit)** 🙏🏼 

.... and credentials **for the bucket** added **into** VM Runtime to securely access the private image.

Comment (apeabody) [reply extras/anthos-vmruntime/README.md]: SG

**Review by Shabirmean — APPROVED** [review]
LGTM!

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.67.87.254**](http://34.67.87.254)
- **Embedded H2** DB backed deployment: [****](http://)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.67.87.254**](http://34.67.87.254)
- **Embedded H2** DB backed deployment: [****](http://)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.67.87.254**](http://34.67.87.254)
- **Embedded H2** DB backed deployment: [**35.226.230.23**](http://35.226.230.23)

**Comment by Shabirmean** [comment]
:zap:  Two deployments have been created for the Point-of-Sale application. You may access and test them at: :zap: </br>
- **MySQL DB** backed deployment: [**34.67.87.254**](http://34.67.87.254)
- **Embedded H2** DB backed deployment: [**35.226.230.23**](http://35.226.230.23)

**Pull request merged**